### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Vivek205**
https://github.com/singnet/refactored-dapp/pull/177

### CODE REVIEW

File: src/assets/thirdPartyServices/snet/coreference_resolution_service/index.js [hunk 1]
```diff
@@ -61,12 +63,12 @@ export default class CoreferenceResolutionService extends React.Component {
     return this.state.methodName !== "Select a method" && this.state.sentence !== "";
   }
 
-  renderServiceMethodNames(serviceMethodNames) {
-    const serviceNameOptions = ["Select a method", ...serviceMethodNames];
-    return serviceNameOptions.map((serviceMethodName, index) => {
-      return <option key={index}>{serviceMethodName}</option>;
-    });
-  }
+  // renderServiceMethodNames(serviceMethodNames) {
```
Comment (chetankothari) [inline src/assets/thirdPartyServices/snet/coreference_resolution_service/index.js:66]: We can delete the code.

File: src/assets/thirdPartyServices/snet/coreference_resolution_service/index.js [hunk 2]
```diff
@@ -94,13 +96,39 @@ export default class CoreferenceResolutionService extends React.Component {
...
     this.setState({ sentence: event.target.value });
   }
 
   renderForm() {
-    const service = this.props.protoSpec.findServiceByName(this.state.serviceName);
-    const serviceMethodNames = service.methodNames;
+
+    const serviceNameOptions = ["Select a method", ...this.props.serviceClient.getMethodNames(ResolveReference)];
+    // const service = this.props.protoSpec.findServiceByName(this.state.serviceName);
+    // const serviceMethodNames = service.methodNames;
```
Comment (chetankothari) [inline src/assets/thirdPartyServices/snet/coreference_resolution_service/index.js:131]: Delete commented code.

File: src/assets/thirdPartyServices/snet/emotion_recognition_service/EmotionVisualizer.js [hunk 3]
```diff
@@ -76,7 +76,8 @@ export default class EmotionVisualizer extends React.Component {
     cnvs.height = img.naturalHeight * scale;
 
     let context = cnvs.getContext("2d");
-    this.props.jobResult["faces"].forEach(item => {
+    //this.props.jobResult["faces"].forEach(item => {
```
Comment (chetankothari) [inline src/assets/thirdPartyServices/snet/emotion_recognition_service/EmotionVisualizer.js:79]: Delete commented code.

File: src/assets/thirdPartyServices/snet/emotion_recognition_service/index.js [hunk 4]
```diff
@@ -1,20 +1,25 @@
...
+  uploadedImage: null,
+  uploadedImageType: null,
+};
 
 export default class EmotionRecognitionService extends React.Component {
   constructor(props) {
     super(props);
     this.handleImageUpload = this.handleImageUpload.bind(this);
-    this.handleServiceName = this.handleServiceName.bind(this);
+    //this.handleServiceName = this.handleServiceName.bind(this);
```
Comment (chetankothari) [inline src/assets/thirdPartyServices/snet/emotion_recognition_service/index.js:16]: Delete commented code.

File: src/assets/thirdPartyServices/snet/emotion_recognition_service/index.js [hunk 5]
```diff
@@ -36,30 +41,58 @@ export default class EmotionRecognitionService extends React.Component {
     });
   }
 
-  handleServiceName(event) {
-    let strService = event.target.value;
-    this.setState({
-      serviceName: strService,
-    });
-  }
+  // handleServiceName(event) {
```
Comment (chetankothari) [inline src/assets/thirdPartyServices/snet/emotion_recognition_service/index.js:44]: Delete commented code.

**Review by chetankothari — APPROVED** [review]
There are a lot of places where we have commented code. We can delete them. Rest looks good.

**Comment by raamb** [comment]
Merging this in now to allow for testing of the UI, we will have a new PR to clean up commented out code across the repo

**Pull request merged**